### DISCUSSION TITLE
Business Report's Accessible via API

### DISCUSSION THREAD

**Discussion by cpiggott** [amzn/selling-partner-api-models#632] [Ideas]
Why is it that Business Report's still aren't accessible via API? Looking through the MWS Discussion Boards this has been something people have been requesting for a long time. 2015/2016?

Knowing things like the Buy Box percentage and other metrics within these reports is super valuable to us, and I know we can download and upload them to our internal reporting service, but it's not scaleable. 

Is there any insight from the Amazon team as to why we've been requesting this for 5 years now and nothing has been done about it?

**Comment by kpconnell** [comment]
I would suggest NOT providing the reports themselves, but an API that supports the detail behind the measures that are on the report - and in this case, buy box percent is already available from other aspects of the API (but page visits is not, and would be nice)

**Reply by cpiggott** [reply]
Yes, I agree. As a report it would be nice, as an API it would be even nicer. But the fact that this still isn't supported, years after it has been requested is a little upsetting. 

It makes you wonder if there is a reason that it's not available. I know that it is available via GET_VENDOR_SALES_DIAGNOSTIC_REPORT if you are a Vendor Central user, but we are not.

**Comment by PhilBroeck** [comment]
I totally support the request for BRs being accessible via API, and I am especially interested in Page Views and Sessions on child ASIN level that are currently only available as csv download from ASC-->Reports-->Business Reports-->By ASIN-->Detail page sales and traffic by child item. This would allow us to evaluate changes to content and prices on ASIN level so much better.
